draft-ietf-ice-rfc5245bis.xml

```diff
@@ -3220,23 +2934,6 @@ in <xref target="RFC5766"/>.
 
 </section>
 
-<section title="Procedures for All Implementations">
-
-<t> ICE has interactions with jitter buffer adaptation mechanisms. An
-RTP stream can begin using one candidate, and switch to another one,
-though this happens rarely with ICE. The newer candidate may result in
-RTP packets taking a different path through the network -- one with
-different delay characteristics. As discussed below, agents are
-encouraged to re-adjust jitter buffers when there are changes in
-source or destination address of media packets. Furthermore, many
-audio codecs use the marker bit to signal the beginning of a
-talkspurt, for the purposes of jitter buffer adaptation. For such
-codecs, it is RECOMMENDED that the sender set the marker bit <xref
-target="RFC3550"/> when an agent switches transmission of media from
-one candidate pair to another.  </t>
-
```

Comment:
Perhaps this could be re-written but we have to have this advice in some RFC but unless we can point at this advice in other specs, I think we need to keep it here.